design/ceph/ceph-2-more-crds.md

```diff
@@ -0,0 +1,139 @@
+# Ceph 2 (More) CRDs
+
+## Use Case
```

Comment:
From the use case description, I'm understanding that the goal is to make the OSDs (or other daemons) more manageable and to more easily see their status? I'm struggling to see the significant benefit of this change since it would require a v2 release and really be a big breaking change. 

The krew plugin and the `storageClassDeviceSets` in the cephcluster CR seem like they already are covering the same scenarios, or could be enhanced to cover the scenarios without any breaking change. For example:
- Krew plugin command can be added to show status of the OSDs (or other daemons)
- Krew plugin commands could be implemented for creating new OSDs by updating the CephCluster CR for you, if you prefer command line parameters instead of yaml

Reply:
Another thought is that if we're going to invest in manageability, we could instead invest in the Ceph dashboard integration, and keep the existing CRDs with no breaking changes since they are an implementation detail for the dashboard config.